web/src/features/public-api/types/score-configs.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  jsonSchema,
+  paginationMetaResponseZod,
+  paginationZod,
+} from "@langfuse/shared";
+import { z } from "zod";
+
+const isPresent = <T>(value: T): value is NonNullable<T> =>
+  value !== null && value !== undefined;
```

Comment:
Do we have some place in the codebase for generic helpers? I'd like to move this function there, I think it's helpful in general. For example if a numeric 0 is a valid input, a falsy check would invalidate it 🤷

Reply:
yes, utils